src/main/java/io/iron/ironworker/client/Client.java

```diff
@@ -278,11 +279,16 @@ public void reload(ScheduleEntity entity) throws APIException {
     private String loadPayload(Map<String, Object> params, Map<String, Object> options) throws GeneralSecurityException, IOException {
         String payload = gson.toJson(params);
         if (options != null) {
-            String encryptionKeyFile = (String) options
-                    .get("encryptionKeyFile");
+            String encryptionKeyFile = (String) options.get("encryptionKeyFile");
             if (encryptionKeyFile != null) {
-                PayloadEncryptor payloadEncryptor = new PayloadEncryptor(encryptionKeyFile);
-                payload = payloadEncryptor.encryptPayload(payload);
+                PayloadEncryptor payloadEncryptor = new PayloadEncryptor(new File(encryptionKeyFile));
+                payload = payloadEncryptor.encrypt(payload);
+            } else {
+                String encryptionKey = (String) options.get("encryptionKey");
```

Comment:
i think you can move this line before `if` block and then use `else if (encryptionKey != null)` instead of 
```
else {
String encryptionKey = (String) options.get("encryptionKey");
if (encryptionKey != null)
```